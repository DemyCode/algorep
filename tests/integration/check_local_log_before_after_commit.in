1
3
no
print_local_log 2
print_local_log 3
print_local_log 4
entry 1 test
start 1
wait_finish 1
sleep 350
print_local_log 2
print_local_log 3
print_local_log 4
stop 1
stop 2
stop 3
stop 4