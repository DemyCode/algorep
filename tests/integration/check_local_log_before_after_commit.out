2
3
4
2 "test"
3 "test"
4 "test"
